tests/data/private_key.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
+MIIEpAIBAAKCAQEAz9wpIlkQ+ICas/e+31cwmqH6P+PfwVGFN67fDFSQjKUpx0+2
+njPeSg20GImOLOpj3/XDv+iYx725kO8TzU554Ua7gNAVi++vJF8OaxE4o5pQP1iC
+TG5C4MMbRmn2WJ929aNpqklisSuvdCsp8p1L+7MncWm455OJyVcQuJxwKdVSPF/w
+2sCF2wU87uJLT5gVys64kTWY7ZPg9WSJGiEKJz9Vqc7HupIP+sFtzoTuQR8AOHwi
+DcBowc6q9JpgTea8vZwSKX3+PLlzVfYFSCIEEc8C4DYFhX51zh0u+/tJ9M9AbZlG
+rcngIURMKhnDKAOWor1GrR3+axG2gio691khjQIDAQABAoIBAGhBTeRrdeaeYnnX
+zkWpLriwe5smCWmmIMJQFJHTrHjOsojEg50P+PN9shIIlGYU3ilL1HXiDRdKItP+
+BhmYvervFkbCl7wLaPY+MzYieh0c3hkXpKr/Ns4TLhAAP84tlXYZE3uUuUqR1bvX
+ln+Yw66oKiVpWJPWXWq+H6Xo2R0xeDPuhBOnlF++glqKsFDCwBI/2ghDNqnmB1pN
+VUMl/NxwdY8vgV9nBA3mb8Zlez+5qWZWeoBDnVX4ZXsGSWwvklkcJuQfnWsLHGgU
+ROKSbYmjFe6jps9exqX4BP0aSZalQH3u/dB3UHQiCiAXik6agsj+4lgqw7t3I6En
+eOC66QUCgYEA/tLYt5vX7LjZDQ7gvnYAF5EfGBrwEHvDPW3JgAgRRhoi0rQq6UoG
+cGoA7rWtpmS2U99ZRuChm+TKzldM80OuGIYJUWkhz7aotiEyR8V/UXR00o/cOmsR
+VlL31x2xGQyOxOHWjHIEGervXa+1RQxR3IVuzjyY6M3qJKjBtZqUN68CgYEA0NHP
+3sVfFbtDOyLXrtNa4bycbVksdMVr+gdW4MmDpMlqWFSnSJNS3FQGYYL7W4opCU5V
+yoMLbUDIHcoD6U1IsDyJwfzQhdKALKK36RngXsSxSRb2zdwqDWoxpg0h8ekUIvSt
+VfONoYzwQPm+rrxG31Jach+AD9tTzBH4zTKPTYMCgYAzjt2iaLnLf6sK5VO7tYxX
+OFmeOvUPoLrkFu6KJRe5o1ggYau8/+lFnW0K319/Jyls1uftW03XiPcvwKIlJjB3
+bcc83ZSlvkNsdlMCCoe0J2froofgmgEsAiV7NNq8QrowQ7FxQTBrPgmoXwDCM7fs
+cmL0/3XAxMj3Gd33rIsIqQKBgQCEUr/vgZCcW7rhqVWVUhc//l39qMlUAZpHqQjz
+jp6d+IgsCxCqG7khJxmYBwEy45zXac1Rc7L0+10UC2RpHXEXRHVi+MpGUrqoDkCO
+NLiKNQfGsgeZMY9wSr/FYoMFIVaCRBvm4Fv6aC4Cbn/XiDU85JmtwapL4gb6xNSt
+awGF8wKBgQC/vfP18yg8ocNJ3CQEr0kPt/ljqHqLQ1+gB0uT1QzY/69sCbKQOg/N
+vxiH1YufA3sYc4I1WKxYtBYvtNA8ht31Z7gHiQnVbqjRKVuzfEGRr72rzzgTyY9t
+JtG3I2e8NPuH56U+n4xfENyTkE3U2oi1V/TeMGcbFR/aT1YzVlii8g==
+-----END RSA PRIVATE KEY-----
```

Comment:
Is this for unit test or actually use to connect to Snowflake?

In the latter case, if the given public key is already registered, anybody can access our db.

Reply:
No this is only for unit tests. And it is testing that this specific private key generate the public key I expect. 

For integration tests, I generate key pair every time in tests.

Reply:
ok